File: Python/taxfree.py
prices = input().split(",")
total = 0
for a in range(len(prices)):
  prices[a] = float(prices[a])

for b in prices:
  if b < 20:
    b += b * 7 / 100
    total += b
  else:
    total += b

print(f"{total:.2f}")

File: Ruby/taxfree.rb
prices = gets.chomp.split(",")
total = 0
for a in (0...prices.length)
  prices[a] = prices[a].to_f
end
for b in prices
  if b <= 20
    b += b * 7 / 100
    total += b
  else
    total += b
  end
end

puts '%.2f' % total